server/playground/index.js

```diff
@@ -0,0 +1,28 @@
+const buildSchema = require("data-sync-gql-core");
```

Comment:
Since this will be the first npm package for sync we may consider using `@aerogear` org for it. `@aerogear/data-sync-gql-core`

@darahayes Opinions?

Reply:
I'm +1 on using `@aerogear`